### DISCUSSION TITLE
Effects using kade engine.

### DISCUSSION THREAD

**Discussion by glowsoony** [KadeArchive/Kade-Engine#2443] [Q&A]
I want to know how to make the screen partly dark like in Lights-down vs imposter and full dark i use modcharts so that would help
example: partly dark 
![image](https://user-images.githubusercontent.com/84847356/135964591-0f6558ea-50f9-4be7-8676-cb0342b828e8.png)

then full dark: 
![image](https://user-images.githubusercontent.com/84847356/135964691-a116fdd6-05ba-4aa3-a8d0-082858ac7403.png)
if any one helps with modchart code on how to do this would be help ful :D

**Comment by bwkam** [comment]
You could download the source code, and see the modchart.lua file, and you see how it's made (it's a .txt file I guess)

**Reply by glowsoony** [reply]
Wrong vs impostor dose not use a modchart for the effects its hard coded but dose not work cause unknown value
